src/iovolpiano.cpp

```diff
@@ -103,10 +104,10 @@ bool VolpianoInput::Import(const std::string &volpiano)
             }
             layer->AddChild(note);
         }
-        else if (ch == 'y' || ch == 'i' || ch == 'z') {
+        else if (ch == 'i' || ch == 'w' || ch == 'x' || ch == 'y' || ch == 'z') {
```

Comment:
maybe make this a static set and use `contains` method from C++20?

Reply:
Since this fits on one single line I think it is okay to leave it like this.